Dockerfile

```diff
@@ -11,11 +11,20 @@ RUN apt-get update \
     && apt-get -qy upgrade --fix-missing --no-install-recommends \
     && apt-get -qy install --fix-missing --no-install-recommends \
         curl \
+        git \
```

Comment:
And what about joining forces across services?  Things like adding git and friends are already covered in Invenio's upstream web node provisioning recipes...